coding-guides/tracking-used-invites.md

```diff
@@ -60,19 +85,17 @@ So now that we have our `invites` object, we're ready to listen to the `guildMem
 ```javascript
 client.on("guildMemberAdd", member => {
   // To compare, we need to load the current invite list.
-  member.guild.invites.fetch().then(guildInvites => {
+  member.guild.invites.fetch().then(newInvites => {
     // This is the *existing* invites for the guild.
-    const ei = invites[member.guild.id];
-    // Update the cached invites for the guild.
-    invites[member.guild.id] = guildInvites;
+    const oldInvites = invites.get(member.guildId);
     // Look through the invites, find the one for which the uses went up.
-    const invite = guildInvites.find(i => ei.get(i.code).uses < i.uses);
+    const invite = newInvites.find(i => i.uses > oldInvites.get(i.code));
     // This is just to simplify the message being sent below (inviter doesn't have a tag property)
     const inviter = client.users.cache.get(invite.inviter.id);
```

Comment:
I would recommend adding a simple check here to see if the invite was not found and then send a message that the user joined but the bot couldn't figure out who invited them.

Reply:
:D